src/views/UserPage/UserPage.vue

```diff
@@ -65,7 +65,14 @@
 
 					<!-- Display Owned Emotes -->
 					<h3 v-if="user && user.owned_emotes?.length" section-title>
-						Owned Emotes ({{ ownedPager.length }})
+						<span>Owned Emotes ({{ ownedPager.length }})</span>
+						<div selector="search-bar">
```

Comment:
This `div` tag isn't permitted in an `h3`:

https://developer.mozilla.org/docs/Web/HTML/Element/Heading_Elements &
https://developer.mozilla.org/docs/Web/Guide/HTML/Content_categories#phrasing_content